packages/radix/src/components/Text.tsx

```diff
@@ -104,8 +105,8 @@ export const Text = styled('span')<TextProps>(
   }),
   variant({
     fontWeight: {
-      normal: { fontWeight: 400, fontFamily: 'normal' },
-      bold: { fontWeight: 500, fontFamily: 'medium' },
+      normal: { fontWeight: 400 },
+      bold: { fontWeight: 500 },
```

Comment:
I'm wondering whether `fontWeight` makes sense to be a Custom Variant now that we're only changing that one property.

Perhaps we can now just rely on the `fontWeight` Style Function?

If we do:
- What happens if the user does `<Text fontWeight="bold">`, will it use the 500 weight?
- What happens if the user does `<Text fontWeight={600}>` (or any number above 500), will it use the 500 weight?
- What happens if the user does `<Text fontWeight={300}>` (or any number below 400), will it use the 400 weight?

Cheers

Reply:
You're right. I'll change it later today 👍

Reply:
I've exposed `fontWeight` from the `radix-system` package. I think it makes sense, but let me know if you were thinking in another approach.

But now every `Text` and `Heading` component with `fontWeight="bold"` look bad because it should be `fontWeight={500}`. Not sure about how you want to handle it. Should I change everywhere `fontWeight="bold"` for `fontWeight={500}`?

Also, should the `fontWeights` property be added to the theme with the possible values (400 and 500)?

Reply:
Thanks. Hmmm yeah sorry maybe it wasn’t clear but i meant before we go ahead to expose font weight it would be good to know the answer to the scenarios above. 

I can do some testing tomorrow, unless you wanna give it a go?

Reply:
My bad 😅

You can see here https://amazing-dubinsky-c304dd.netlify.com/docs/text/ an example with _all_ possible `font-weight` values. The rendered weight really depends on the font-face and its weights being loaded.

Reply:
Right, ok.

I think we need to think about the best Component API here. 

Maybe the best thing to do, for now, is keep the `bold` variant, so we can manually map it to `500`.

The work you have done is already an improvement, since we no longer need to specify the `font-family`. Thanks for that.

Reply:
Ok then! Unnecessary changes reverted and the variant is kept.